src/utils/localStorage.ts

```diff
@@ -0,0 +1,8 @@
+export function saveStateToLocalStorage(key: string, state: boolean) {
+    localStorage.setItem(key, JSON.stringify(state));
+}
+
+export function loadStateFromLocalStorage(key: string): string | null {
+    const storedState = localStorage.getItem(key);
+    return storedState ? JSON.parse(storedState) : null;
+}
```

Comment:
there is something similar to this already in utils/functions. I like what you did here but remove the function from the other place just for this to be the only place for it to be. Also `saveStateToLocalStorage` should receive a state that is unknown. That way it can work for other use cases.

Reply:
Fixed! I like the unknown for the state type 👍🏻